src/main/java/ladysnake/dissolution/common/config/DissolutionConfig.java

```diff
@@ -68,6 +68,9 @@
         @Config.Comment("If set to false, incorporeal players won't be able to use the /dissolution stuck command to get back to their spawnpoint")
         public boolean allowStuckCommand = true;
 
+        @Config.Comment("If set to true, will kill you when out of experience. Not fully tested.")
```

Comment:
I'd rather you fully test the change before pushing an update.

Reply:
Whelp back to building it because while waiting I tested it under varius scenarios...